test/integration/spec/room.js

```diff
@@ -156,6 +158,50 @@ describe('Room', function() {
     });
   });
 
+  [true, false].forEach(simulcastEnabled => {
+    (isChrome ? describe : describe.skip)(`getStats with VP8 Simulcast ${simulcastEnabled ? 'enabled' : 'not enabled'}`, function() {
```

Comment:
```suggestion
    (isChrome && isSafari ? describe : describe.skip)(`getStats with VP8 Simulcast ${simulcastEnabled ? 'enabled' : 'not enabled'}`, function() {
```

Reply:
I think you meant (isChrome || isSafari) - I will update.